ext/console/01_console.js

```diff
@@ -898,6 +899,54 @@ function formatRaw(ctx, value, recurseTimes, typedArray, proxyDetails) {
             return ctx.stylize(base, "date");
           }
         }
+      } else if (
+        typeof globalThis.Temporal !== "undefined" &&
```

Comment:
It looks like this needs to be updated to match what's done in https://github.com/denoland/deno/pull/21669/files#diff-e994540878ba98beb4e9623acf5d8023b06b2b9ed0a383fde3256e13cdd991cd

Reply:
I'm unable to see what this link leads too - could you paste the relevant code here?

Reply:
See the recent `proxyDetails === null` checks in the other if statements surrounding this code

Reply:
I did `Proxy` support to get the test code through, but honestly it's nonsense and might as well be removed. Wrapping anything other than `Array` and `Error` in `Proxy` makes most methods unusable.

```js
new Proxy(new Set(), {}).add(1);
// => throws TypeError: Method Set.prototype.add called on incompatible receiver #<Set>
```

Reply:
```suggestion
        proxyDetail === null &&
        typeof globalThis.Temporal !== "undefined" &&
```

It would seem that simply adding `proxyDetail === null` would be sufficient